ko_KR/readme.md

```diff
@@ -0,0 +1,971 @@
+
+*기여하는 방법에 대해 찾아보고 있다면 (이슈/PR/etc), 우리의 [기여 가이드](contributing.md)를 확인하세요.*
+
+[AVA 트위터 계정](https://twitter.com/ava__js)을 팔로우하면 업데이트에 대한 소식을 받아볼 수 있습니다.
+
+전체 번역 문서들: [Español](https://github.com/sindresorhus/ava-docs/blob/master/es_ES/readme.md), [Français](https://github.com/sindresorhus/ava-docs/blob/master/fr_FR/readme.md), [Italiano](https://github.com/sindresorhus/ava-docs/blob/master/it_IT/readme.md), [日本語](https://github.com/sindresorhus/ava-docs/blob/master/ja_JP/readme.md), [Português](https://github.com/sindresorhus/ava-docs/blob/master/pt_BR/readme.md), [Русский](https://github.com/sindresorhus/ava-docs/blob/master/ru_RU/readme.md), [简体中文](https://github.com/sindresorhus/ava-docs/blob/master/zh_CN/readme.md), [한국어](https://github.com/sindresorhus/ava-docs/blob/master/ko_KR/readme.md)
+
+## 목차
+
+- [사용법](#사용법)
+- [CLI 사용법](#cli-사용법)
+- [구성](#구성)
+- [참조 문서](#참조-문서)
+- [API](#api)
+- [가정](#가정)
+- [도움말](#도움말)
+- [FAQ](#faq)
+- [레시피](#레시피)
+- [지원](#지원)
+- [관련된 프로젝트](#관련된-프로젝트)
+- [링크](#링크)
+- [팀](#팀)
+
+## 왜 AVA인가요?
+
+- 작고 빠릅니다
+- 간단한 테스트 문법
+- 테스트를 동시적으로 실행
+- 작은 테스트를 작성하도록 유도
+- 내재적인 전역 변수 없음
+- [각 테스트 파일마다 고립된 환경](#process-isolation)
+- [ES2015로 테스트 작성](#es2015-support)
+- [Promise 지원](#promise-support)
+- [생성기 함수 지원](#generator-function-support)
+- [비동기 함수 지원](#async-function-support)
+- [Observable 지원](#observable-support)
+- [강화된 가정 메시지](#enhanced-assertion-messages)
+- [선택적인 TAP 출력](#optional-tap-output)
+- [깨끗한 스텍 기록](#clean-stack-traces)
+
+## 테스트 문법
+
+```js
+import test from 'ava';
+
+test(t => {
+	t.deepEqual([1, 2], [1, 2]);
+});
+```
+
+## 사용법
+
+### AVA를 프로젝트에 추가하기
+
+AVA를 전역에 설치한 후 `--init` 인수와 함께 실행하면 `package.json`에 AVA를 추가할 수 있습니다:
+
+```console
+$ npm install --global ava
+$ ava --init
+```
+
+```json
+{
+  "name": "awesome-package",
+  "scripts": {
+    "test": "ava"
+  },
+  "devDependencies": {
+    "ava": "^0.11.0"
+  }
+}
+```
+
+`--init` 뒤에 전달된 인수는 `package.json`에 추가됩니다.
+
+#### 수동 설치
+
+물론 AVA를 직접 설치할 수도 있습니다:
+
+```console
+$ npm install --save-dev ava
+```
+
+`package.json`의 `test` 스크립트에서 `ava`를 사용하도록 설정해야 합니다.
+(위를 참고하세요)
+
+### 테스트 파일 만들기
+
+프로젝트 루트 디렉터리에 `test.js` 파일을 하나 생성합니다:
+
+```js
+import test from 'ava';
+
+test('foo', t => {
+	t.pass();
+});
+
+test('bar', async t => {
+	const bar = Promise.resolve('bar');
+
+	t.is(await bar, 'bar');
+});
+```
+
+<img src="https://github.com/sindresorhus/ava/blob/master/screenshot.png" width="150" align="right">
+
+### 실행하기
+
+```console
+$ npm test
+```
+
+### 감시 모드로 실행하기
+
+```console
+$ npm test -- --watch
+```
+
+AVA는 지능형 감시 모드와 함께 제공됩니다. [이 레시피에 대해 자세히 알아보기](docs/recipes/watch-mode.md).
```

Comment:
Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/watch-mode.md)`